job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -173,6 +180,23 @@ String kogitoWithSpecDroolsNightlyStage(String kogitoVersion, String kogitoBranc
     """
 }
 
+String serverlessLogicNightlyStage(String productVersion, String droolsVersion, String branch, String configBranch, String nexusSuffix = getNexusFromVersion(productVersion)) {
+    return """
+        stage('trigger Serverless Logic nightly job ${productVersion}') {
+            steps {
+                build job: 'kogito.nightly/${branch}', propagate: false, wait: true, parameters: [
+                        [\$class: 'StringParameterValue', name: 'DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-kogito-${nexusSuffix}/content-compressed'],
+                        [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: '${getUMBFromVersion(productVersion)}'],
```

Comment:
The UMB message needs to be changed so we do not send the same message by two jobs - or is it expected that kogito 1.24 would be only for serverless logic?

Reply:
yes, and some future followed up releases too

Reply:
`-blue` suffix added to `UMB_VERSION`